### PR TITLE
Add getbans API

### PR DESCRIPTION
**Pull request by zeptin**
Sample output:

```
[
  {
    "endPoint": "[::ffff:89.xxx.xxx.xxx]:16178",
    "banUntil": "2019-06-27T21:33:34.0218677Z",
    "banReason": "Banned via the API."
  }
]
```

### CODE REVIEW

**Comment by Neurosploit** [comment]
@zeptin can you add this api test as well to NetworkControllerApiTests.cs?

  [Fact]
        public async Task Can_GetBannedPeers_From_ApiAsync()
        {
            using (var builder = NodeBuilder.Create(this))
            {
                var network = new StratisRegTest();

                var nodeA = builder.CreateStratisPosNode(network, "nc-2-nodeA").Start();

                var nodeB_Ip = "127.0.0.2";
                var nodeB_IpAddress = IPAddress.Parse(nodeB_Ip);
                var nodeB_EndPoint = new IPEndPoint(nodeB_IpAddress, 0);

                var nodeB_Ip2 = "127.0.0.3";
                var nodeB_IpAddress2 = IPAddress.Parse(nodeB_Ip2);
                var nodeB_EndPoint2 = new IPEndPoint(nodeB_IpAddress2, 0);

                var nodeAaddressManager = nodeA.FullNode.NodeService<IPeerAddressManager>();
                nodeAaddressManager.AddPeer(new IPEndPoint(nodeB_IpAddress, 0), IPAddress.Loopback);

                var peerBanning = nodeA.FullNode.NodeService<IPeerBanning>();
                peerBanning.BanAndDisconnectPeer(nodeB_EndPoint, "This peer is banned because of a reason");
                Assert.True(peerBanning.IsBanned(nodeB_EndPoint));

                peerBanning.BanAndDisconnectPeer(nodeB_EndPoint2, "This peer is banned because of another reason");
                Assert.True(peerBanning.IsBanned(nodeB_EndPoint2));

                var bannedPeers = await $"http://localhost:{nodeA.ApiPort}/api".AppendPathSegment("network/getbans").GetJsonAsync<List<BannedPeerModel>>();

                Assert.Equal(2, bannedPeers.Count);
                bannedPeers = bannedPeers.OrderBy(b => b.EndPoint).ToList();

                var bannedPeer = bannedPeers[0];
                Assert.Equal("This peer is banned because of a reason", bannedPeer.BanReason);
                Assert.True(DateTime.UtcNow < bannedPeer.BanUntil.Value);
                Assert.Equal("[::ffff:127.0.0.2]:0", bannedPeer.EndPoint);

                bannedPeer = bannedPeers[1];
                Assert.Equal("This peer is banned because of another reason", bannedPeer.BanReason);
                Assert.True(DateTime.UtcNow < bannedPeer.BanUntil.Value);
                Assert.Equal("[::ffff:127.0.0.3]:0", bannedPeer.EndPoint);
            }
        }

File: src/Stratis.Bitcoin/Controllers/Models/BannedPeerModel.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using Stratis.Bitcoin.Controllers.Converters;
+
+namespace Stratis.Bitcoin.Controllers.Models
+{
+    /// <summary>
+    /// Class representing a banned peer.
+    /// </summary>
+    public class BannedPeerModel
+    {
+        public BannedPeerModel()
```
Comment (fassadlr) [inline src/Stratis.Bitcoin/Controllers/Models/BannedPeerModel.cs]: Don't need this constructor :)

Comment (zeptin) [reply src/Stratis.Bitcoin/Controllers/Models/BannedPeerModel.cs]: Fixed

**Comment by zeptin** [comment]
https://dev.azure.com/Stratisplatformuk/StratisBitcoinFullNode/_workitems/edit/4059

**Pull request merged**